### PR TITLE
Update README.es-mx.md

### PR DESCRIPTION
**Pull request by J-cordz**
This Pull Request improves the content on the `es-mx` translation, it changes some words for better understanding, fixes typos and adds spacing when needed. 

Graduación-GitHub-2022

### CODE REVIEW

**Review by github-education — DISMISSED** [review]
**Hi J-cordz,**
**Welcome to graduation!**


### I have a few items I need you to take care of before I can merge this PR:

* *I'm not seeing a valid student developer pack approval, please submit an [application](https://education.github.com/discount_requests/student_application?utm_source=2021-06-05-GitHubGraduation) to get started*
* *It looks like you still need to fill out the [shipping form](https://airtable.com/shrVMo8ItH4wjsO9f) to continue*
* *Uh Oh! I've found some issues with where you have created your files!* 
	*The required markdown file does not exist, please ensure the file `_data/J-cordz/J-cordz.md` exists*
*Please ensure all changes are contained within the `_data/J-cordz/` directory. Invalid file paths:* 

	* `translations/README.es-mx.md`

* *You need to agree to our COC on the shipping form pretty please!*

Feel free to re-request a review from me and I'll come back and take a look!

File: translations/README.es-mx.md [hunk 1]
```diff
@@ -1,42 +1,38 @@
...
  ⚠️ el formulario tiene que completarse antes de crear tu Pull Request (PR) y no garantiza la participación en el evento. Tu PR debe integrarse exitosamente en el repositorio y sólo los primeros 7,500 PRs fusionados recibirán tarjetas en el correo.
 2. **Sube una pull request** con tu información de perfil para unirte al Anuario y ser destacado en el evento de graduación.
 
 ## 1. Completa el formulario de envío.
- Información subida al [formulario de envio de swag](https://airtable.com/shrVMo8ItH4wjsO9f) sólo se utiliza para enviar las tarjetas de comercio para la graduación. Completar el formulario no garantiza que recibas algo en el correo. Sólo los primeros 7,500 graduados que fusionen su Pull Request para el Anuario de GitHub van a recibir un envío.
+La información subida al [formulario de envio de swag](https://airtable.com/shrVMo8ItH4wjsO9f) sólo se utiliza para enviar las tarjetas de coleccionables para la graduación. Completar el formulario no garantiza que recibas algo en el correo. Sólo los primeros 7,500 graduados que fusionen su Pull Request para el Anuario de GitHub van a recibir un envío.
```
Comment (juanpflores) [inline translations/README.es-mx.md:33]:
```suggestion
La información subida al [formulario de envio de swag](https://airtable.com/shrVMo8ItH4wjsO9f) sólo se utiliza para enviar las tarjetas de colección para la graduación. Completar el formulario no garantiza que recibas algo en el correo. Sólo los primeros 7,500 graduados que fusionen su Pull Request para el Anuario de GitHub van a recibir un envío.

```

**Review by juanpflores — APPROVED** [review]

**Comment by juanpflores** [comment]
Gracias por ayudarnos a mejorar la traducción ✨

**Comment by J-cordz** [comment]
De nada @juanpflores! Me alegra mucho el poder ayudar! :octocat: 🎉

**Pull request merged**